Doppler.HtmlEditorApi/Infrastructure/IDummyRepository.cs

```diff
@@ -0,0 +1,12 @@
+using Doppler.HtmlEditorApi.Model;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Doppler.HtmlEditorApi.Infrastructure
+{
+    public interface IDummyRepository
+    {
+        ContentModel GetCampaignModel(string accountName, int campaignId);
+        TemplateModel GetTemplateModel(string accountName, int templateId);
+        TemplateModel GetSharedTemplateModel(int templateId);
```

Comment:
The real implementation will be asynchronous, I think that it is a good idea to prepare it:

```suggestion
        Task<ContentModel> GetCampaignModel(string accountName, int campaignId);
        Task<TemplateModel> GetTemplateModel(string accountName, int templateId);
        Task<TemplateModel> GetSharedTemplateModel(int templateId);
```

Reply:
Updated